js/multi.js

```diff
@@ -0,0 +1,440 @@
+    }
+
+    async fetchTradingFees (params = {}) {
+        await this.loadMarkets ();
+        const response = await this.publicGetFeeSchedules (params);
+        const fees = [];
+        for (let i = 0; i < response.length; i++) {
+            const fee = response[i];
+            fees.push ({
+                'minVolume': fee['minVolume'],
+                'maker': fee['makerFee'],
+                'taker': fee['takerFee'],
+            });
+        }
+        return {
+            'info': response,
+            'fees': fees,
+        };
+    }
+
+    async fetchTrades (symbol, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'market': market['id'],
+        };
+        if (limit !== undefined) {
+            request['limit'] = limit;
+        }
+        const response = await this.publicGetMarketTrade (this.extend (request, params));
+        return this.parseTrades (response['result'], market, since, limit);
+    }
+
+    parseTrade (trade, market) {
+        const symbol = market['symbol'];
+        const timestamp = this.safeTimestamp (trade, 'time');
+        const price = this.safeFloat (trade, 'price');
+        const amount = this.safeFloat (trade, 'amount');
+        return {
+            'info': trade,
+            'id': this.safeString (trade, 'id'),
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'symbol': symbol,
+            'side': trade['type'],
+            'price': price,
+            'amount': amount,
+            'cost': parseFloat (price * amount),
+            'order': undefined,
+            'takerOrMaker': undefined,
+            'type': undefined,
+            'fee': undefined,
+        };
+    }
+
+    async fetchTicker (symbol, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const marketId = market['id'];
+        const response = await this.publicGetMarketStatusAll (params);
+        const marketTicket = this.getMarketTicket (response, marketId);
+        return this.parseTicker (marketTicket['result'], symbol);
+    }
+
+    getMarketTicket (response, marketId) {
+        const marketTicker = {};
+        for (let i = 0; i < response.length; i++) {
+            if (response[i]['market'] === marketId) {
+                marketTicker['result'] = response[i];
+                break;
+            }
+        }
+        return marketTicker;
+    }
+
+    parseTicker (ticker, symbol) {
+        return {
+            'symbol': symbol,
+            'info': ticker,
+            'high': ticker['high'],
+            'low': ticker['low'],
+            'bid': ticker['bid'],
+            'bidVolume': ticker['volume'],
+            'ask': ticker['ask'],
+            'open': ticker['open'],
+            'close': ticker['close'],
+            'last': ticker['close'],
```

Comment:
This should most likely be mark or alternatively last. It might be worth opening an issue. Mark price is a quite recent addition that gives more of an insight of current market offering.